user/lxc/template.py

```diff
@@ -0,0 +1,44 @@
+pkgname = "lxc"
+pkgver = "6.0.1"
+pkgrel = 0
+build_style = "meson"
+configure_args = [
+    "-Dapparmor=false",
+    "-Dman=false",
+    "-Dinit-script=sysvinit",
+    "-Ddistrosysconfdir=default",
+]
+hostmakedepends = ["git", "meson", "pkgconf"]
```

Comment:
git doesn't seem to be needed
```suggestion
hostmakedepends = ["meson", "pkgconf"]
```